evcouplings/align/tools.py

```diff
@@ -10,6 +10,304 @@
 from evcouplings.utils.system import (
     run, create_prefix_folders, verify_resources, temp
 )
+from evcouplings.utils.config import check_required
+
+
+from os import path
```

Comment:
for importing: first import os, then the evcouplings tools (convention)
No blank lines between things being imported